eisbuk-admin/cypress/plugins/firebasePlugin/updateFirestore/adminDb.ts

```diff
@@ -0,0 +1,6 @@
+import admin from "firebase-admin";
+
+process.env["FIRESTORE_EMULATOR_HOST"] = "localhost:8080";
```

Comment:
I think it would be better to set this value when invoking the tests. Or at least leave an already set value there.
Something like:

```typescript
process.env["FIRESTORE_EMULATOR_HOST"] = process.env["FIRESTORE_EMULATOR_HOST"] || "localhost:8080";
```